template-manager-cmtemplate/src/test/resources/output/kafka-without-hdfs.bp

```diff
@@ -0,0 +1,126 @@
+{
+  "cdhVersion": "7.0.1",
+  "displayName": "CDP HA Streaming Cluster",
+  "cmVersion": "7.x.0",
+  "services": [
+    {
+      "refName": "streams_messaging_manager",
+      "serviceType": "STREAMS_MESSAGING_MANAGER",
+      "serviceConfigs": [
+        {
+          "name": "zookeeper_service",
+          "ref": "zookeeper"
+        }
+      ],
+      "roleConfigGroups": [
+        {
+          "refName": "streams_messaging_manager-STREAMS_MESSAGING_MANAGER_SERVER-BASE",
+          "roleType": "STREAMS_MESSAGING_MANAGER_SERVER",
+          "base": true
+        },
+        {
+          "refName": "streams_messaging_manager-STREAMS_MESSAGING_MANAGER_UI-BASE",
+          "roleType": "STREAMS_MESSAGING_MANAGER_UI",
+          "base": true
+        }
+      ]
+    },
+    {
+      "refName": "kafka",
+      "serviceType": "KAFKA",
+      "serviceConfigs": [
+        {
+          "name": "zookeeper_service",
+          "ref": "zookeeper"
+        }
+      ],
+      "roleConfigGroups": [
+        {
+          "refName": "kafka-GATEWAY-BASE",
+          "roleType": "GATEWAY",
+          "base": true
+        },
+        {
+          "refName": "kafka-KAFKA_BROKER-BASE",
+          "roleType": "KAFKA_BROKER",
+          "base": true
+        }
+      ]
+    },
+    {
+      "refName": "zookeeper",
+      "serviceType": "ZOOKEEPER",
+      "roleConfigGroups": [
+        {
+          "refName": "zookeeper-SERVER-BASE",
+          "roleType": "SERVER",
+          "base": true
+        }
+      ]
+    },
+    {
+      "refName": "schemaregistry",
+      "serviceType": "SCHEMAREGISTRY",
+      "serviceConfigs": [
+        {
+          "name": "zookeeper_service",
+          "ref": "zookeeper"
+        }
+      ],
+      "roleConfigGroups": [
+        {
+          "refName": "schemaregistry-SCHEMA_REGISTRY_SERVER-BASE",
+          "roleType": "SCHEMA_REGISTRY_SERVER",
+          "base": true
+        }
+      ]
+    },
+    {
+      "refName": "core_settings",
+      "serviceType": "CORE_SETTINGS",
+      "serviceConfigs": [
+        {
+          "name": "core.defaultFs",
```

Comment:
Based on https://jira.cloudera.com/secure/attachment/231501/231501_out.diff (referenced from the Jira of this PR) the name should be different! I.e. `core_defaultfs`